### PR TITLE
Show diff table

### PR DESCRIPTION
**Pull request by zachyam**
Added /state page which shows the overall state of the modules across different projects

### CODE REVIEW

File: kaknas/static/js/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        success: function (data) {
+          if (data == 'False') {
+            moduleRefCell.style.color = 'red';
+          } else if (data == 'True') {
+            moduleRefCell.style.color = 'black';
+          }
+        },
+    });
+}
+function populateModuleInfo(module_state_map) {
```
Comment (psalaberria002) [inline kaknas/static/js/index.js]: Could you try to refactor the app so there is no reference to equinor, or europe-west1-1? We could have 100s of projects, this won't scale.

File: kaknas/static/js/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    var modulePathCell = document.createElement('td');
+    var modulePathList = document.createElement('ul');
+    for (var module_path in module_info) {
+      var singleModulePath = document.createElement('li');
+      singleModulePath.textContent = module_path;
+      modulePathList.append(singleModulePath);
+      modulePathCell.append(modulePathList);
+      moduleRow.append(modulePathCell);
+    }
+    for (var module_path in module_info) {
```
Comment (psalaberria002) [inline kaknas/static/js/index.js]: Quite a lot of repetition going on, please refactor it

File: manager.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
-from kaknas import create_app
+from kaknas.app import app
 
 if __name__ == "__main__":
-    app = create_app()
+    # app = create_app()
```
Comment (psalaberria002) [inline manager.py]: Why commented out? If you don't need it, delete it

**Comment by psalaberria002** [comment]
Sorry I don't have time for a thorough review, maybe @eyvind can have a look ;)

File: kaknas/static/js/index.js (hunk 1 above)
Comment (zachyam) [reply kaknas/static/js/index.js]: This is for a quick proof of concept. Will definitely refactor once I have a better grasp of project goals, hopefully after planning meeting this week.

File: kaknas/static/js/index.js (hunk 2 above)
Comment (zachyam) [reply kaknas/static/js/index.js]: Same as reply above

**Pull request merged**